src/Apps/MyCollection/Routes/EditArtwork/Utils/categoryOptions.ts

```diff
@@ -26,4 +26,5 @@ export const categoryOptions = [
   { text: "Ephemera or Merchandise", value: "Ephemera or Merchandise" },
   { text: "Reproduction", value: "Reproduction" },
   { text: "NFT", value: "NFT" },
+  { text: "Digital Art", value: "Digital Art" },
```

Comment:
**suggestion:** Looks good, but mind bumping L25 to the bottom as we did elsewhere?